tests/unit/test_main.py

```diff
@@ -122,7 +122,57 @@ async def test_create_mw_case_4(self, event_loop):
 
         assert response.status_code == 400
         assert current_data['description'] == \
-                         'End before start not allowed'
+                         'end: End before start not allowed'
+        self.scheduler.add.assert_not_called()
+
+    # async def test_create_mw_case_5(self, event_loop):
```

Comment:
@Ktmi was this test case commented out intentionally or just temporarily as you were making changes?

Reply:
Commented out intentionally, as it tests the behavior of not allowing ID to be set, which is planned for v2.

Reply:
I see. @Ktmi, on `pytest` there's a [`pytest.mark.skip(reason="some_reason")` decorator](https://github.com/kytos-ng/kytos-end-to-end-tests/blob/master/tests/test_e2e_13_mef_eline.py#L184), that's handy for such cases, we've been using on other repos.

Reply:
Thanks, didn't know that was an option.